manifests/velero.yaml.tpl

```diff
@@ -96,13 +95,12 @@ spec:
     backupsEnabled: true
     snapshotsEnabled: true
 
-    deployRestic: true
+    # deployRestic: true
     # will be replace by 
-    # deployNodeAgent: true
-    # nodeAgent:
-    restic:
+    deployNodeAgent: true
+    nodeAgent:
       podVolumePath: /var/lib/kubelet/pods
-      privileged: false
+      privileged: true
```

Comment:
I believe this one can be removed:
https://github.com/vmware-tanzu/helm-charts/blob/dde9531816fe1750828bb903a5a44d11bb4c1148/charts/velero/README.md?plain=1#L94